packages/lib-classifier/src/store/Step.spec.js

```diff
@@ -104,4 +104,56 @@ describe('Model > Step', function () {
       })
     })
   })
+
+  describe('with a next step', function () {
+    it('should have isThereANextStep return true', function () {
+      const step = Step.create({ stepKey: 'S1', taskKeys: ['T1', 'T2'], next: 'S2' })
+      expect(step.isThereANextStep).to.be.true()
+    })
+  })
+
+  describe('with a previous step', function () {
+    it('should have isThereAPreviousStep return true', function () {
+      const step = Step.create({ stepKey: 'S2', taskKeys: ['T1', 'T2'], previous: 'S1' })
+      expect(step.isThereAPreviousStep).to.be.true()
+    })
+  })
+
+  describe('with a single choice branching task', function () {
+    let tasks
+    before(function () {
+      tasks = [
+        MultipleChoiceTask.TaskModel.create(MultipleChoiceTaskFactory.build({ taskKey: 'T1', required: false })),
+        // SingleChoiceTaskFactory defaults to a branching single choice task (answers with different next values)
+        SingleChoiceTask.TaskModel.create(SingleChoiceTaskFactory.build({ taskKey: 'T2', required: false }))
+      ]
+    })
+
+    it('should have isThereBranching return true', function () {
+      const step = Step.create({ stepKey: 'S1', taskKeys: ['T1', 'T2'], tasks })
+      expect(step.isThereBranching).to.be.true()
+    })
+  })
```

Comment:
I included the above examples and a test expecting `isThereBranching` to be false if all answers' `next` is not defined, so several failing tests as desired.

Subsequent commit refactors `isThereBranching` resulting in passing tests.

Reply:
This looks great. Could we add one more test that tests for backwards compatibility where the `next` property is set to a task like `'T2'` rather than a step? This will test the conversion to steps route.